modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/XrefProcess_conf.pm

```diff
@@ -0,0 +1,367 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::XrefProcess_conf;
+
+use strict;
+use warnings;
+
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::Base_conf');
+
+use Bio::EnsEMBL::Hive::Version 2.5;
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+
+sub default_options {
+  my ($self) = @_;
+
+  return {
+    %{$self->SUPER::default_options()},
+    'release'          => $self->o('ensembl_release'),
```

Comment:
Hi Marc. The use of 'ensembl_release' here defaults back to Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf where the Ensembl API version is retrieved if the -release option isn't specified in the xref run. This has been the case even before this split project, and I worry that changing it would affect people running the pipeline as they didn't need to worry about setting the ENS_VERSION env variable before. Not to say that it can't be done, just that we should discuss the need for it.

Reply:
Forget it, it will be fine as such, we'll update if needed later.